README.md

```diff
@@ -858,9 +858,9 @@ Extra Eclipse plugins
 
 * GWT plugin
 
-    * [Download and install the Eclipse GWT plugin](http://code.google.com/intl/en/eclipse/docs/getting_started.html)
+    * [Download and install the Eclipse GWT plugin](http://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html)
 
-        * Note: it is recommended to keep your Eclipse GWT plugin version in sync with the GWT version that we use.
+        * Note: it is recommended to use GWT SDK version in Eclipse GWT plugin GWT Settings preference in sync with the GWT version that we use. Look for `version.com.google.gwt` property value in [droolsjbpm-build-bootstrap/pom.xml](pom.xml)
```

Comment:
Sorry, not sure this makes sense "it is recommended to use GWT SDK version in Eclipse GWT plugin GWT Settings preference in sync with the GWT version that we use" perhaps "it is recommended to keep the GWT SDK version in Eclipse's GWT plugin GWT Settings preference in sync with the GWT version that we use". Just a minor comment.. thanks for the PR :-)

Reply:
np, maybe I did more than I would. I will try another change ;)